components/push/android/src/main/java/mozilla/appservices/push/PushManager.kt

```diff
@@ -199,10 +174,11 @@ class PushManager(
  * Please contact services back-end for any additional bridge protocols.
  */
 
+@Suppress("Unused")
 enum class BridgeType {
     FCM, ADM, APNS, TEST;
 
-    override fun toString() = name.toLowerCase()
+    override fun toString() = name.toLowerCase(Locale.US)
```

Comment:
I'm kind of fascinated to hear the story behind this locale thing, and whether we should document it for future reference (since I don't see any existing references to `Locale.US` elsewhere in the codebase).

Reply:
It's something that I learned from an Android Studio warning:
Here's some literature about it: https://garygregory.wordpress.com/2015/11/03/java-lowercase-conversion-turkey/

TL;DR: `toLowerCase` will use the user locale and it may lead to incorrect results. Turkish language for example will transform `I` to `ı` (Latin Small Letter Dotless I).

Reply:
I believe it's a lint rule that complains if you don't specific a locale. `Locale.ROOT` might work or maybe `Locale.getDefault()`.